TFNetworkRecLayer.py

```diff
@@ -5955,3 +5955,72 @@ def _get_output_native_rec_op(self, cell):
       y = y[0]
 
     return y
+
+
+class ZoneoutLSTMCell(BaseRNNCell):
+  '''Wrapper for tf LSTM to create Zoneout LSTM Cell
+
+  inspired by:
+  https://github.com/teganmaharaj/zoneout/blob/master/zoneout_tensorflow.py
+
+  Published by one of 'https://arxiv.org/pdf/1606.01305.pdf' paper writers.
+
+  Many thanks to @Ondal90 for pointing this out. You sir are a hero!
+  '''
+
+  def __init__(self, num_units, zoneout_factor_cell=0., zoneout_factor_output=0., state_is_tuple=True, name=None, **kwargs):
+    '''Initializer with possibility to set different zoneout values for cell/hidden states.
+    '''
```

Comment:
Use `"""`. Keep the style consistent.
Also, please document all arguments, including specifying their types (see other code for example).